### DISCUSSION TITLE
Docking hatch wiring

### DISCUSSION THREAD

**Discussion by SAYtan47** [FakeFishGames/Barotrauma#14571] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

For ships like remora, camel, orca2 and R-29 lower docking hatch isnt connected to the power grid which means these ships cannot get power through it, while for ships like kastrull, orca, typhon and typhon2 lower docking hatch is connected to the grid. To solve this inconsistancy might as well just connect all the docking hatches to the power grid, there is room in junction boxes for it on every mentioned ship.

![20240904223832_1](https://github.com/user-attachments/assets/a08471e8-35b0-4f86-908b-cbe5dc692828)
![20240904223917_1](https://github.com/user-attachments/assets/799d7672-0878-4aa8-910a-9fd984692dd6)
![20240904223949_1](https://github.com/user-attachments/assets/9f8b3d91-9f40-4fc1-8969-298e44182f2f)
![20240904224030_1](https://github.com/user-attachments/assets/a6a01a5e-d5ec-4e49-bf0a-88529b86dd91)
w
![20240904224108_1](https://github.com/user-attachments/assets/e45855fc-7e37-487c-97fd-6b893e4f1372)
![20240903172032_1](https://github.com/user-attachments/assets/560fd139-ddda-4a30-b1a5-be60c1c2a850)
![20240904223704_1](https://github.com/user-attachments/assets/8e21b1ff-8a8c-4704-a625-f6ee26972142)
![20240904223742_1](https://github.com/user-attachments/assets/a508cfbb-aee5-4d1b-b63f-5396965ce83b)

### Reproduction steps

1. Go to the sub editor
2. Pick one of the mentioned subs and enable wiring mode
3. Check the lower docking hatch on each one

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/bbc4ea831dac31ae8a68d235127377d08228dd90 (commit in our private repo), coming up in the next unstable build.
